libraries/node-core-library/src/SubprocessTerminator.ts

```diff
@@ -1,14 +1,24 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import { Executable } from '@rushstack/node-core-library';
 import * as child_process from 'child_process';
 import process from 'process';
 
+import { Executable } from './Executable';
+
 /**
  * Details about how the `child_process.ChildProcess` was created.
+ *
+ * @beta
  */
 export interface ISubprocessOptions {
+  /**
+   * Whether or not the child process is detached.
+   *
+   * @remarks
+   * On POSIX systems, detached=true is required for killing the subtree. On Windows, detached=true
+   * creates a separate console window which is generally not the desired behavior.
```

Comment:
>On Windows, detached=true creates a separate console window which is generally not the desired behavior.

Can the process be killed on Windows without `detatched = true`? Should we change this behavior to not open a new window on Windows but still be killable?

Reply:
Talked offline, this is an option specifying the current state of the child process, not an arg used for creating a new process. I updated the comment to be more useful.